src/vendors/oasisscan.ts

```diff
@@ -50,7 +53,16 @@ export function getOasisscanAPIs(url: string | 'https://api.oasisscan.com/mainne
     return data
   }
 
-  async function getTransactionsList(params: { accountId: string; limit: number }): Promise<Transaction[]> {
+  async function getTransaction({ hash }: { hash: string }) {
```

Comment:
TODO: Add cache mechanism, as transaction won't ever change once we already fetch it.

Reply:
https://github.com/oasisprotocol/oasis-wallet-web/pull/1976